examples/text_labels_and_annotations/angle_annotation.py

```diff
@@ -185,7 +185,7 @@ def update_text(self):
         r = s / 2
         if self.textposition == "inside":
             r = s / np.interp(angle_span, [60, 90, 135, 180],
-                                          [3.3, 3.5, 3.8, 4])
+                                          [10, 3.5, 11.4, 4])
```

Comment:
I'm not sure this is the right approach to fix the issue. This is moving the text, but the issue is really about the scaling of the angle markings.